src/uniprotkb/components/entry/__tests__/SimilarProteins.spec.tsx

```diff
@@ -1,37 +1,39 @@
-import renderWithRouter from '../../../../shared/__test-helpers__/RenderWithRouter';
-import SimilarProteins from '../SimilarProteins';
+import { act } from 'react-dom/test-utils';
+import { fireEvent } from '@testing-library/react';
+import renderWithRedux from '../../../../shared/__test-helpers__/RenderWithRedux';
+import SimilarProteins from '../similar-proteins/SimilarProteins';
 import similarProteinsData from './__mocks__/similarProteinsData.json';
+import accessionsData from './__mocks__/accessionsData.json';
 
-jest.mock('../../../../shared/hooks/useDataApi', () => jest.fn());
-import useDataApi from '../../../../shared/hooks/useDataApi';
-import { fireEvent } from '@testing-library/react';
+var axios = require('axios');
+var MockAdapter = require('axios-mock-adapter');
 
-const dataMock = {
-  loading: false,
-  data: similarProteinsData,
-};
+var axiosMock = new MockAdapter(axios);
+axiosMock.onGet(/\/uniref\/search/).reply(200, similarProteinsData);
+axiosMock.onGet(/\/uniprotkb\/accessions/).reply(200, accessionsData);
 
-const getRendered = () =>
-  renderWithRouter(
-    <SimilarProteins
-      primaryAccession="P05067"
-      isoforms={{ isoforms: ['P05067-4'] }}
-    />
-  );
-describe('SimilarProteins tests', () => {
-  beforeEach(() => {
-    useDataApi.mockImplementation(() => dataMock);
+const getRendered = async () => {
```

Comment:
This should probably be moved inside a `beforeEach` and have it rendered to a variable in scope of all the tests, in order to have the same pattern than most of the other tests have in the codebase.